### PR TITLE
Add 3rd party licenses file and update doc

### PR DESCRIPTION
**Pull request by gzussa**
### What does this PR do?

Add 2rd party file

### Verification Process

manual review

### Review checklist (to be filled by reviewers)

- [ ] Feature or bug fix MUST have appropriate tests (unit, integration, etc...)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -26,10 +33,21 @@ This project does not have a strict release schedule. However, we would make a r
...
+In order to create a new release you will need to setup PyPI Tooling as well as `datadog_checks_dev` mentioned above.
+See our internal documentation for more details.
+
+### Release
+
+1. Tag the release on GitHub. [Example](https://github.com/DataDog/datadogpy/releases/tag/v0.10.0)
+2. Checkout the [tag release](#commands).
+3. Run `ddev release build .` and `ddev release upload --sdist`.
   - Make sure that both an `sdist` and a [universal wheel](https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels) have been uploaded to [PyPI](https://pypi.python.org/pypi/datadog/).
+4. Set CHANGELOG release date. [Example](https://github.com/DataDog/datadogpy/commit/e89b19c0bc1e5ea2ec026e14b11d05b980062ffb)
```
Comment (nmuesch) [inline RELEASING.md]: Can this be part of the changelog creation process? If the release is done in a short timeframe, we could just have one PR 😄

Comment (zippolyte) [inline RELEASING.md]: `ddev` already sets the date
```suggestion
```

File: RELEASING.md [hunk 2]
```diff
@@ -26,10 +33,21 @@ This project does not have a strict release schedule. However, we would make a r
...
+#### Setup PyPI
+
+In order to create a new release you will need to setup PyPI Tooling as well as `datadog_checks_dev` mentioned above.
+See our internal documentation for more details.
+
+### Release
+
+1. Tag the release on GitHub. [Example](https://github.com/DataDog/datadogpy/releases/tag/v0.10.0)
+2. Checkout the [tag release](#commands).
+3. Run `ddev release build .` and `ddev release upload --sdist`.
```
Comment (zippolyte) [inline RELEASING.md]:
```suggestion
3. Run `ddev release build .` and `ddev release upload --sdist .`
```

File: RELEASING.md [hunk 3]
```diff
@@ -26,10 +33,21 @@ This project does not have a strict release schedule. However, we would make a r
...
+
+#### Setup PyPI
+
+In order to create a new release you will need to setup PyPI Tooling as well as `datadog_checks_dev` mentioned above.
+See our internal documentation for more details.
+
+### Release
+
+1. Tag the release on GitHub. [Example](https://github.com/DataDog/datadogpy/releases/tag/v0.10.0)
+2. Checkout the [tag release](#commands).
```
Comment (zippolyte) [inline RELEASING.md]:
```suggestion
2. Checkout the tag created at the previous step.
```

File: RELEASING.md [hunk 4]
```diff
@@ -26,10 +33,21 @@ This project does not have a strict release schedule. However, we would make a r
...
+### Prerequisite
+
+#### Setup PyPI
+
+In order to create a new release you will need to setup PyPI Tooling as well as `datadog_checks_dev` mentioned above.
+See our internal documentation for more details.
+
+### Release
+
+1. Tag the release on GitHub. [Example](https://github.com/DataDog/datadogpy/releases/tag/v0.10.0)
```
Comment (zippolyte) [inline RELEASING.md]:
```suggestion
1. Create the release on GitHub. [Example](https://github.com/DataDog/datadogpy/releases/tag/v0.33.0)
```

File: RELEASING.md [hunk 5]
```diff
@@ -10,9 +10,16 @@ This project does not have a strict release schedule. However, we would make a r
...
+* Check and upgrade dependencies where it applies and makes sense. [Example](https://github.com/DataDog/datadogpy/commit/f81efe8cbf6e5bc5cb4ab46da750248161d0c548#diff-2eeaed663bd0d25b7e608891384b7298)
+  - Create a distinct pull request and test your changes since it may introduce regressions.
+  - While using the latest versions of dependencies is advised, it may not always be possible due to potential compatibility issues.
+  - Upgraded dependencies should be thoroughly considered and tested to ensure they are safe!
 * Make sure tests are passing.
+  - Locally and in the continuous integration system.
 * Make sure documentation is up-to-date and building correctly.
 * Build the package locally (e.g. `python3 setup.py sdist`), install it into a fresh virtualenv and try playing around with it for a bit.
+* [Update changelog](#update-changelog)
+  - Create a distinct pull request.
```
Comment (zippolyte) [inline RELEASING.md]: Let's remove this item from here.

**Pull request merged**